skyportal/handlers/api/observation_plan.py

```diff
@@ -200,3 +201,42 @@ def delete(self, observation_plan_request_id):
         )
 
         return self.success()
+
+
+class ObservationPlanDownloadHandler(BaseHandler):
+    @auth_or_token
+    def get(self, observation_plan_request_id):
+        """
+        ---
+        description: Get a json dictionary representation of the observation plan.
+        tags:
+          - observation_plan_requests
+        parameters:
+          - in: path
+            name: observation_plan_id
+            required: true
+            schema:
+              type: string
+        responses:
+          200:
+            content:
+              application/json:
+                schema: SingleObservationPlanRequest
+        """
+
+        options = [
+            joinedload(ObservationPlanRequest.observation_plans)
+            .joinedload(EventObservationPlan.planned_observations)
+            .joinedload(PlannedObservation.field)
+        ]
+
+        observation_plan_request = ObservationPlanRequest.get_if_accessible_by(
+            observation_plan_request_id,
+            self.current_user,
+            mode="read",
+            raise_if_none=True,
+            options=options,
+        )
+        self.verify_and_commit()
+
+        return self.success(data=observation_plan_request)
```

Comment:
What's the difference between this and the regular `ObservationPlanRequest` handler?

Reply:
@guynir42 Great point!